tasks/install_tasks.py

```diff
@@ -71,6 +73,39 @@ def install_shellcheck(ctx, version="0.8.0", destination="/usr/local/bin"):
     ctx.run(f"rm -rf \"/tmp/shellcheck-v{version}\"")
 
 
+@task
+def install_protoc(ctx, version="26.1", destination="~/go/bin"):
+    """
+    Installs the requested version of protoc in the specified folder (by default /usr/local/bin).
+    Required generate the golang code based on .prod (inv generate-protobuf).
+    """
+
+    if sys.platform == 'win32':
+        print("protoc is not supported on Windows")
+        raise Exit(code=1)
+    if sys.platform.startswith('darwin'):
+        platform_os = "osx"
+    if sys.platform.startswith('linux'):
+        platform_os = "linux"
+
+    match platform.machine().lower():
+        case "aarch64":
+            platform_arch = "aarch_64"
+        case "arm64":
+            platform_arch = "x86_64"
```

Comment:
Should this case be `amd64`?

Moreover, I think some cases need to be added. For instance, https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/594268559 failed because `platform.machine().lower()` was `x86_64`, which is supported.

Reply:
yes your are right, I made a copy/past type :( 

PR updated https://github.com/DataDog/datadog-agent/pull/28166/commits/64adb1ebf0e5fa8de91518a5884e2dd47fe6c4b8